Demo/Demo.entitlements

```diff
@@ -2,6 +2,10 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>com.apple.developer.associated-domains</key>
+	<array>
+		<string>applinks:braintree-ios-demo.fly.dev</string>
```

Comment:
just curious, why do we add this here

Reply:
Great question - so this is adding a new entitlement for associated domains to our demo app. [Associated domains](https://developer.apple.com/documentation/xcode/supporting-associated-domains) establish a secure association between domains and our app. It essentially establishes a secure connection between the server we hosted our app association file at and our demo app while also establishing the permissions of that connection (in our case we are just allowing app links/universal links). The app association file on the server and associated domain entitlement have to match exactly to work as expected.